app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java

```diff
@@ -302,8 +302,7 @@ private void populatePlaces(List<Place> placeList) {
 
         if (placeList.size() == 0) {
             int duration = Toast.LENGTH_SHORT;
-            Toast toast = Toast.makeText(this, R.string.no_nearby, duration);
-            toast.show();
+            ViewUtil.showSnackbar(findViewById(R.id.container), R.string.no_nearby);
```

Comment:
duration variable is never used, if you remove that codacy will be happy @ujjwalagrawal17 :)